examples/jaxrs-json-provider-jettison/README_es.adoc

```diff
@@ -0,0 +1,155 @@
+= JAX-RS Proveedor JSON con Jettison
+:index-group: REST
+:jbake-type: page
+:jbake-status: status=published
+
+Este es un ejemplo de como configurar en TomEE 7.x o posterior el proveedor antiguo de JSON, Jettison, usado por TomEE1.7.x.
+
+
+Este escenario es útil cuando las aplicaciones REST son migradas de TomEE1.7.x hacia TomEE1.x o posterior y se desea continuar con la antigua salida JSON de Jettison 1.3.7.
+
+
+=== Corre las pruebas
+Este ejemplo contiene 2 casos de prueba, uno usando Jettison y otro sin él para fines comparativos.
+
+[source,java]
+----
+mvn clean test
+----
+
+
+=== Habilitación de Jettison como proveedor JSON
+
+Debe proporcionar los siguientes 2 archivos en su carpeta WEB-INF:
+
+* openejb-jar.xml
+[source,xml]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<openejb-jar xmlns="http://www.openejb.org/openejb-jar/1.1">
+    <pojo-deployment class-name="org.superbiz.JAXRSApplication">
+        <properties>
+            cxf.jaxrs.providers = json
+        </properties>
+    </pojo-deployment>
+</openejb-jar>
+
+----
+
+* resources.xml
+
+[source,java]
+----
+
+<?xml version="1.0" encoding="UTF-8"?>
+<resources>
+    <Service id="json" class-name="org.apache.cxf.jaxrs.provider.json.JSONProvider">
+        SkipJaxbChecks = true
+        DropRootElement = false
+        SupportUnwrapped = true
+        SingleJaxbContext = true
+    </Service>
+</resources>
+
+----
+
+Y finalmente asegúrese de que Jettison JAR 1.3.7 esté disponible en la carpeta TomEE / lib.
```

Comment:
I know this is just a translation from the original `README.afoc`, but this observation about "JAR 1.3.7 is available in TomEE /lib folder" sounds like a case where you are not using maven

Reply:
Hello @emecas  thanks for the corrections, I thought the same thing, but i guess the example has to work for people not using maven.